tfhe/src/typed_api/booleans/tests.rs

```diff
@@ -0,0 +1,154 @@
+use std::ops::{BitAnd, BitOr, BitXor};
+
+use crate::typed_api::prelude::*;
+use crate::typed_api::{generate_keys, set_server_key, ClientKey, ConfigBuilder, FheBool, FheBoolParameters, CompressedFheBool};
+
+fn setup_static_default() -> ClientKey {
+    let config = ConfigBuilder::all_disabled().enable_default_bool().build();
+
+    let (my_keys, server_keys) = generate_keys(config);
+
+    set_server_key(server_keys);
+    my_keys
+}
+
+fn setup_static_tfhe() -> ClientKey {
+    let config = ConfigBuilder::all_disabled()
+        .enable_custom_bool(FheBoolParameters::tfhe_lib())
+        .build();
+
+    let (my_keys, server_keys) = generate_keys(config);
+
+    set_server_key(server_keys);
+    my_keys
+}
+
+#[test]
+fn test_xor_truth_table_static_default() {
+    let keys = setup_static_default();
+
+    let ttrue = FheBool::encrypt(true, &keys);
+    let ffalse = FheBool::encrypt(false, &keys);
```

Comment:
question why do we need to explicitely encrypt with some client keys but the server keys are hidden/managed for the user ?

Reply:
Because in my opinon, making the keys explicit in the encryption process is better than hidding them.

It makes it clear that you are encrypting / decrypting data, it makes it clear which key you are using.
Also it makes it clear that you are the owner of the these keys, and its kinda crucial as they must stay private.

I feel that if there where a `set_client_key` / `set_public_key`, then you lose the "encrypting is done with keys", and
it might also create confusion: "what is the lib doing with these keys, I hope it does not leak them".

Hidding the fact that server keys are necessary to write operations on encrypted data by managing them for the user
is great because it hides details that is not so imporant for this API level, and allows to write simpler code by using common operators.

I don't think hidding the "encryption is done with client / public keys" really brings something beneficial